app/views/items/edit_stock.html.erb

```diff
@@ -62,8 +62,8 @@
       <tbody>
         <% @item.versions.reverse.each do |version| %>
           <tr>
-            <td><%= version.created_at.strftime("%b-%d-%Y %l:%M%P") %></td>
-            <td><%= version.whodunnit %></td>
+            <td><%= local_time(version.created_at, "%b-%d-%Y %l:%M%P") %></td>
+            <td><%= user_name(version.whodunnit) %></td>
```

Comment:
If we are saving whodunit as user_id why don't we call is user_id? was this from the gem it's self. I just think it could be a less ambiguous reference